config/desktop/focal/environments/i3-wm/config_base/packages

```diff
@@ -63,6 +64,7 @@ i3
 i3lock
 i3status
 i3-wm
+suckless-tools
```

Comment:
Is this a requirement for i3-wm?

Reply:
yes as the suckless-tools package includes packages needed to use i3-wm properly, like dmenu used for app/command execution